boot/camlheader

```diff
@@ -0,0 +1 @@
+#!/usr/local/bin/ocamlrun
```

Comment:
This file shouldn't be here (I expect it's been committed thanks to `git add -A` or `git add .`)